packages/eslint-config-mc-app/helpers/rules-presets.js

```diff
@@ -0,0 +1,293 @@
+/**
+ * This file contains rules copied from other libraries, to make it easier to differentiate with
+ * our own rules and to better maintain them.
+ */
+
+// The ESLint browser environment defines all browser globals as valid,
+// even though most people don't know some of them exist (e.g. `name` or `status`).
+// This is dangerous as it hides accidentally undefined variables.
+// We blacklist the globals that we deem potentially confusing.
```

Comment:
We enforce a deny list of the globals...

Reply:
Yeah thanks, noticed that too.